### DISCUSSION TITLE
Fonts

### DISCUSSION THREAD

**Discussion by mya1919** [jeffaroundtheworld/2024gamecollab#35] [Q&A]
we need two fonts, one for titles and one for the basic writing.
I thought that since it was based in a school they shouldn't be too much of generic horror fonts.
Let me know what you guys think about these ones. 

for titles:
![image](https://github.com/jeffaroundtheworld/2024gamecollab/assets/159198555/cd739d43-cfb9-4d8b-b42a-7c8852112caa)

for basic writing:
![image](https://github.com/jeffaroundtheworld/2024gamecollab/assets/159198555/f84b1825-0973-41ed-8525-bf906d6ecb94)

**Comment by kodriq** [comment]
I think those r pretty good and we could especially use them on the like start screen
We might need 1 more simple one for dialogue since it being easy to read is all that matters for that, but those r perfect for everything else
(also maybe a typewriter font if we add documents lying around?)

**Comment by kodriq** [comment]
i got a font im using for the dialogue, just gonna put it here since its related 
https://www.1001freefonts.com/splendid-66.font

**Comment by JackV13** [comment]
Where did you find these fonts?
